uberfire-services/uberfire-services-api/src/main/java/org/guvnor/common/services/backend/cache/LRUCache.java

```diff
@@ -32,14 +32,11 @@
     private Map<Path, V> cache;
 
     public LRUCache() {
-        cache = new LinkedHashMap<Path, V>(MAX_ENTRIES + 1,
-                                           0.75f,
-                                           true) {
-            public boolean removeEldestEntry(Map.Entry eldest) {
-                return size() > MAX_ENTRIES;
-            }
-        };
-        cache = (Map) Collections.synchronizedMap(cache);
+        setCache(this.MAX_ENTRIES);
```

Comment:
since it is static, you should use class name (`LRUCache.MAX_ENTRIES`) or just omit `this`

Reply:
Done